pkgs/development/compilers/zig/generic.nix

```diff
@@ -13,7 +13,6 @@
   callPackage,
   version,
   hash,
-  patches ? [ ],
```

Comment:
Why delete this?

Reply:
It's not, there's an `args.patches` reference.

Reply:
Edit: LSPs detect it as unused, however since we didn't have the ellipses at the end of the function args, we can't collect arbitrary other args in {}@args, so I further modified it to add the ellipses.

Reply:
The patches identifier itself was unused, which is what led to the confusion. I'd be cool with either keeping patches or using args.patches, but LSPs and LSP like tools were marking it as unused (even if incorrectly)